fiat-roles/src/main/java/com/netflix/spinnaker/fiat/roles/UserRolesSyncer.java

```diff
@@ -105,7 +106,7 @@ public void onApplicationEvent(RemoteStatusChangedEvent event) {
     isEnabled.set(isInService());
   }
 
-  @Scheduled(fixedDelay = 30000L)
+  @Scheduled(fixedDelayString = "#{userRolesSyncer.getSyncDelayMs()}")
```

Comment:
The log message comes from the Kork library. Was hesitant to change it there since the message might be useful for other services using the lock library.

Agreed that grabbing a lock more frequently than the delay would be ideal though I think the schedule should be calculated based on the syncDelay say every syncDelay/2 ms? What do you think?

This isn't a big issue at all but it just eases debugging Fiat since currently most of the logs are just filled with `could  not acquire lock messages`.